soci/artifacts.go

```diff
@@ -468,6 +481,11 @@ func loadArtifact(artifactBkt *bolt.Bucket, digest string) (*ArtifactEntry, erro
 	if err != nil {
 		return nil, err
 	}
+	encodedSpanSize := artifactBkt.Get(bucketKeySpanSize)
+	spanSize, err := dbutil.DecodeInt(encodedSpanSize)
```

Comment:
`dbutil.DecodeInt` returns an error if the input is nil. AFAICT, this will break for artifacts created before updating SOCI.